python/repositories.bzl

```diff
@@ -255,13 +267,15 @@ py_runtime_pair(
         python_path = python_bin,
         python_version = python_short_version,
         python_version_nodot = python_short_version.replace(".", ""),
+        coverage_tool = rctx.attr.coverage_tool if rctx.attr.coverage_tool == None and "windows" not in rctx.os.name else "\"{}\"".format(rctx.attr.coverage_tool),
```

Comment:
I think this logic has a bug?

If tool=None and os=windows, then it'll do format(None), resulting in `"None"`, which isn't a valid value.

Is this supposed to be "or" instead of "and" ?

Reply:
Thanks, there was a bug, fix it now.